### PR TITLE
okhttp deleted

### PR DESCRIPTION
**Pull request by amiluslu**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13_PREVIEW" default="false" project-jdk-name="13" project-jdk-type="JavaSDK">
```
Comment (barancev) [inline .idea/misc.xml:6]: This change is not related to the PR, remove it from this PR.

File: bazel-amilusSelenium [hunk 2]
```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_tcamiluslu/4ea50276b8925b7e467cb75929e42874/execroot/selenium
```
Comment (barancev) [inline bazel-amilusSelenium:1]: This change is not related to the PR, remove it from this PR.

File: .idea/misc.xml (hunk 1 above)
Comment (amiluslu) [reply .idea/misc.xml]: Yeap, I just deleted this file from PR.

File: bazel-amilusSelenium (hunk 2 above)
Comment (amiluslu) [reply bazel-amilusSelenium]: Yeap, I just deleted this file from PR.

File: java/server/test/org/openqa/selenium/netty/server/NettyServerTest.java [hunk 3]
```diff
@@ -62,7 +61,7 @@ public void ensureMultipleCallsWorkAsExpected() {
       ).start();
 
       // Very deliberately uses OkHttp.
-      HttpClient client = new OkHttpClient.Factory().createClient(server.getUrl());
+      HttpClient client = (HttpClient) HttpClient.Factory.create(server.getUrl().toString());
```
Comment (shs96c) [inline java/server/test/org/openqa/selenium/netty/server/NettyServerTest.java:64]: This cast is wrong. `create` returns a new http client factory (not a client), and we use the string to figure out the name of the factory to use ("okhttp", "netty", "reactor"). The server url gets passed to _that_ in order to create the httpclient itself.

**Comment by shs96c** [comment]
@amiluslu thank you very much for the patch! Welcome to the project!

The current Netty client seems solid enough, and we're working on making the reactor-based one the default. I think it's time to land this. I had to do some manual work to rebase this so it's in as 93dde77b8a4909bde69004507cc859b73ea19cd8
